docs/en/spark/configuration/source-plugins/Cassandra.md

```diff
@@ -0,0 +1,57 @@
+# Source plugin: Cassandra
+
+### Description
+
+Read data from Cassandra.
+
+### Env
+| name           | type   | required | default value |
+| -------------- | ------ | -------- | ------------- |
+| [spark.cassandra.connection.host](#spark.cassandra.connection.host-string)       | string | yes      | -             |
+
+##### spark.cassandra.connection.host [string]
+
+Cassandra connection host
+
+##### other
+
+Refer to [spark-cassandra-connector-options](https://github.com/datastax/spark-cassandra-connector/blob/b2.4/doc/reference.md) for configurations.
+
+### Options
+
+| name             | type   | required | default value |
+| --------------   | ------ | -------- | ------------- |
+| [table](#table-string)            | string | yes      | -             |
+| [keyspace](#keyspace-string)         | string | yes      | -             |
+| [cluster](#cluster-string)          | string | no       | default       |
+| [pushdown](#pushdown-string)         | string | no       | true         |
+
```

Comment:
If I remember correctly `result_table_name` seems to be required, please check, thanks